config/checkstyle_checks.xml

```diff
@@ -110,7 +110,7 @@
     <property name="message" value="Line should not be longer than 100 symbols"/>
   </module>
   <module name="RegexpSingleline">
-    <property name="format" value="^(?!(.*href=&quot;|.*http)).{101,}$"/>
+    <property name="format" value="^(?!(.*href=&quot;[^&quot;]+&quot;&gt;|.*http)).{101,}$"/>
```

Comment:
what about enforcing `<` start the line for the `<a`? If the link is long enough, I think it makes sense it be completely alone on the line.

Reply:
done. Now there is many changes in `google_style.xml`